### PR TITLE
SN449: Add job scheduling to automatic translation endpoints

### PR DESCRIPTION
**Pull request by prakharrathi25**
# Description

This PR converts the automatic translation endpoints to celery async tasks. The PR also creates a new celery container to manage the `low-priority` queue. This is done so that the main tasks in the `projects` and `dataset` app are not affected due to the queue. We can assign separate concurrency and works to both celery queues to manage the task load. New low priority tasks can be routed to the `functions` queue. 

Fixes [#SN449](https://project-sunbird.atlassian.net/browse/SN-449?atlOrigin=eyJpIjoiMWVkYzBiYjhhMjJkNDYxOGE3ZDY5OWE5ZmUxMjMwYzAiLCJwIjoiaiJ9)

## Type of change

Please delete options that are not relevant.

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Please describe the tests / checks that you performed to verify your changes. Provide instructions so we can reproduce. Please delete options that are not relevant.

- [x] Tested the API endpoint on Swagger with the 'Try it out' feature
- [x] Any other tests you performed for checking edge cases.
- [x] Any other tests you performed for checking exceptions.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: backend/functions/tasks.py [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+                else:
+                    translated_sentences = translations_output
+
+            elif api_type == "google":
+                # Get the translation using the Indictrans NMT API
+                translations_output = get_batch_translations_using_google_translate(
+                    sentence_list=batch_of_input_sentences,
+                    target_language=output_language,
+                    checks_for_particular_languages=checks_for_particular_languages,
+                )
```
Comment (GokulNC) [inline backend/functions/tasks.py:133]: Sometimes, Google API fails to auto-detect language. In such cases, it just returns the input sentences as it is. We observed it in a few Shoonya tasks.

Can you please also explicit pass the input_langauge

File: backend/functions/views.py [hunk 2]
```diff
@@ -2,179 +2,174 @@
...
-    return "Success"
+# def sentence_text_translate_and_save_translation_pairs(
+#     languages,
+#     input_dataset_instance_id,
+#     output_dataset_instance_id,
+#     batch_size,
+#     api_type="indic-trans",
+#     checks_for_particular_languages=False,
+# ):
+#     """Function to translate SentenceTexts and to save the TranslationPairs in the database.
```
Comment (GokulNC) [inline backend/functions/views.py:28]: Once the PR is tested, please remove all these comments before merging.

**Review by GokulNC — CHANGES_REQUESTED** [review]

**Review by ishvindersethi22 — APPROVED** [review]
Tested on Dev Server

File: backend/functions/views.py (hunk 2 above)
Comment (prakharrathi25) [reply backend/functions/views.py:28]: Yes yes, I will remove them now.

File: backend/functions/tasks.py [hunk 3]
```diff
@@ -0,0 +1,206 @@
+from email.policy import strict
+import pandas as pd
```
[the remark this answers is not in the document]
Comment (prakharrathi25) [reply backend/functions/tasks.py:2]: I think it creeped in there by mistake. I will remove it.

File: backend/functions/tasks.py (hunk 1 above)
Comment (prakharrathi25) [reply backend/functions/tasks.py]: I have made the change. The `source_language` is being passed explicitly now.

**Review by ishvindersethi22 — APPROVED** [review]

**Review by GokulNC — APPROVED** [review]
Thanks Prakhar, nicely done! :)

**Comment by prakharrathi25** [comment]
> Thanks Prakhar, nicely done! :)

Thanks!
